grpc/src/main/java/com/linecorp/armeria/server/grpc/GrpcServiceBuilder.java

```diff
@@ -562,6 +566,26 @@ public GrpcServiceBuilder useClientTimeoutHeader(boolean useClientTimeoutHeader)
         return this;
     }
 
+    /**
+     * Adds a {@link GrpcHealthCheckService} to this {@link GrpcServiceBuilder}.
+     */
+    public GrpcServiceBuilder addGrpcHealthCheckService(GrpcHealthCheckService grpcHealthCheckService) {
```

Comment:
```suggestion
     * Sets a {@link GrpcHealthCheckService} to this {@link GrpcServiceBuilder}.
     */
    public GrpcServiceBuilder grpcHealthCheckService(GrpcHealthCheckService grpcHealthCheckService) {
```

Reply:
`enableGrpcHealthCheckService()` and `grpcHealthCheckService(...)` are mutually exclusive.
How about raising an `IllegalStateException` if both are set.

Reply:
Could we add `<a href="https://github.com/grpc/grpc/blob/master/doc/health-checking.md">GRPC Health Checking Protocol</a>` to the Javadoc?